scheduler/src/cook/mesos/util.clj

```diff
@@ -342,3 +354,31 @@
    (keyword (name name-space) (name value)))
   ([name-space subspace value]
    (namespace-datomic (str (name name-space) "." (name subspace)) value)))
+
+(defn clear-uncommitted-jobs
```

Comment:
At the moment, no. My plan was to eventually have a thread call it periodically to clean things up. However, I didn't want to add too much to this PR. For the moment this function could be called manually using the repl.